### PR TITLE
Spelling and Punctuation Fixes

### PR DESCRIPTION
**Pull request by klorpa**
#### Summary
Interface "Fixes spelling errors and standardizes punctuation and spelling."

#### Purpose of change
Provides a more consistent interface.

### CODE REVIEW

File: doc/POSTAPOC_PRICE_GUIDE.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ pretty much the most anyone will be willing to spend on any one thing, no matter
 ```
 
 #### Food pricing
-Food is priced mostly for a combination of calories, how preservable they are, and how nutritious they are.
+Food is priced mostly for a combination of calories, how perservable they are, and how nutritious they are.
```
Comment (Night-Pryanik) [inline doc/POSTAPOC_PRICE_GUIDE.md:102]: "preservable" is right here, I suppose. It means "how long it can last while preserved" or something like that.

Comment (PatrikLundell) [reply doc/POSTAPOC_PRICE_GUIDE.md:102]: That's confusing. I don't think "perservable" (the "correction") is a word. Maybe "their (extendable) shelf life" or something like that might be used.

Comment (kevingranade) [reply doc/POSTAPOC_PRICE_GUIDE.md:102]: https://www.thefreedictionary.com/preservable

Comment (kevingranade) [inline doc/POSTAPOC_PRICE_GUIDE.md:102]:
```suggestion
Food is priced mostly for a combination of calories, how preservable they are, and how nutritious they are.
```

Comment (PatrikLundell) [reply doc/POSTAPOC_PRICE_GUIDE.md]: I'm still confused. The reference is to pREservable as far as I can see (and what the link itself says), not the correction's pERservable.

File: data/json/snippets/newspapers.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
       },
       {
         "id": "many_years_old_news_7",
-        "text": "ALIENS AMONG US!  Janine Galfrizowich, of Martha's Vineyard, wrote in to our Paranormal Investigation Staff with this cryptic gem.  \"They're always watching, always watching from the shadows.  Stealing my avocados and watching!  They took my neighbor and made him into one of them!\"  Our journalists are trying to track Mrs Galfrizowich down, but it is clear enough that this ties into the well known Avocado Conspiracy (see issue 24, volume 7)."
+        "text": "ALIENS AMONG US!  Janine Galfrizowich, of Martha's Vineyard, wrote in to our Paranormal Investigation Staff with this cryptic gem.  \"They're always watching, always watching from the shadows.  Stealing my avocados and watching!  They took my neighbor and made him into one of them!\"  Our journalists are trying to track Mrs. Galfrizowich down, but it is clear enough that this ties into the well known Avocado Conspiracy (see issue 24, volume 7)."
```
Comment (kevingranade) [inline data/json/snippets/newspapers.json:32]:
```suggestion
        "text": "ALIENS AMONG US!  Janine Galfrizowich, of Martha's Vineyard, wrote in to our Paranormal Investigation Staff with this cryptic gem.  \"They're always watching, always watching from the shadows.  Stealing my avocados and watching!  They took my neighbor and made him into one of them!\"  Our journalists are trying to track Mrs Galfrizowich down, but it is clear enough that this ties into the well known Avocado Conspiracy (see issue 24, volume 7)."
```

**Pull request merged**